### PR TITLE
Fix line numbers in the tokenizer

### LINKED ISSUE #923: Line numbers are off by one
Cf. the second paragraph of https://github.com/r-lib/roxygen2/issues/664#issuecomment-532434400

### PR DESCRIPTION
**Pull request by gaborcsardi**
Closes #923.

### CODE REVIEW

File: tests/testthat/test-tokenize.R [hunk 1]
```diff
@@ -90,3 +90,61 @@ test_that("Inline comments do not extend past the closing brace", {
...
+test_that("Line numbers are ok", {
+
+  check_line_nums <- function(block, lines) {
+    for (t in names(lines)) {
+      expect_equal(block_get_tag(block, t)$line, lines[[t]], info = t)
+    }
+  }
+
+  text <-
+"#' @title
```
Comment (hadley) [inline tests/testthat/test-tokenize.R]: Can you indent this please? Otherwise looks good to go.

Comment (gaborcsardi) [reply tests/testthat/test-tokenize.R]: Just that line? I am not sure how to indent multi-line text. Maybe we should use glue for this?

Comment (hadley) [reply tests/testthat/test-tokenize.R]: The column number doesn't matter for this code?

Comment (gaborcsardi) [reply tests/testthat/test-tokenize.R]: Right. I guess I could have looked at the rest of the tests.... should be good now....

**Comment by hadley** [comment]
Thanks!

**Pull request merged**